### PR TITLE
Data.Fixed instance

### PR DESCRIPTION
**Pull request by phadej**

### CODE REVIEW

File: src/Data/Binary/Class.hs [hunk 1]
```diff
@@ -239,6 +250,18 @@ instance Binary Integer where
                     let v = roll bytes
                     return $! if sign == (1 :: Word8) then v else - v
 
+-- | /Since: 0.7.x.y/
+#ifdef HAS_FIXED_CONSTRUCTOR
```
Comment (kolmodin) [inline src/Data/Binary/Class.hs]: @phadej Will Data.Fixed have different serialisation formats depending on the branch of this `ifdef`?

Comment (phadej) [inline src/Data/Binary/Class.hs]: No, in both cases `Integer` is serialised. I could add a test which is run when `MkFixed` is available, that they are actually the same.

**Comment by phadej** [comment]
@kolmodin added tests.

**Comment by phadej** [comment]
And actually pushed.

File: src/Data/Binary/Class.hs [hunk 2]
```diff
@@ -251,6 +262,18 @@ instance Binary Integer where
                     let v = roll bytes
                     return $! if sign == (1 :: Word8) then v else - v
 
+-- | /Since: 0.7.x.y/
```
Comment (kolmodin) [inline src/Data/Binary/Class.hs]: Next version will be 0.8.0.0.

**Comment by phadej** [comment]
@kolmodin fixed `Since: version`.

**Comment by phadej** [comment]
and now they pass

**Pull request merged**